packages/docs/components/Input/Input.js

```diff
@@ -0,0 +1,24 @@
+import { Input } from '@platzily-ui/components';
+import { createStyleSheet } from '@platzily-ui/styling';
+
+const useStyleSheet = createStyleSheet((theme) => ({
+  inputStyle: {
+    padding: theme.spacing(2),
+  },
+}), { key: 'padding' });
+
+const InputComponent = (props) => {
+  const { classes } = useStyleSheet();
+  const { required, type, placeholder, disabled, otherProps } = props;
+
+  return <Input
+    className={classes.inputStyle}
```

Comment:
Good use of the useStyleSheet hook :grin:. We could change the classname to `input` instead.